src/miral/symbols.map

```diff
@@ -438,15 +438,19 @@ global:
     miral::WaylandExtensions::supported[*;
     miral::WaylandExtensions::zwlr_layer_shell_v1*;
     miral::WaylandExtensions::zxdg_output_manager_v1*;
+    miral::WindowInfo::attached_edges*;
```

Comment:
Unless we plan on cherry-picking into 1.3.0 (I think that unnecessary) these need moving to 2.7